src/js/view/controls/components/timeslider.js

```diff
@@ -108,9 +108,17 @@ class TimeSlider extends Slider {
 
         // Show the tooltip on while dragging (touch) moving(mouse), or moving over(mouse)
         this.elementUI = new UI(this.el, { useHover: true, useMove: true })
-            .on('drag move over', this.showTimeTooltip.bind(this), this)
-            .on('dragEnd out', this.hideTimeTooltip.bind(this), this)
+            .on('drag move over', this.showTimeTooltip, this)
+            .on('out', this.hideTimeTooltip, this)
             .on('click', () => this.el.focus());
+
+        this.el.addEventListener('focus', (e) => this.updateAriaText(e));
+        this._model.on('seeked', () => {
```

Comment:
Use the model argument, instead of `this._model` inside this callback:

```js
this._model.on('seeked', (model) => {
    if (model.get('scrubbing')) {
```

Reply:
@robwalch this is a trigger, not a change, so the `{type: "seeked"}` is what is being passed into callback, not the model

Reply:
Gotcha. Then just chain this listener with the other model listeners above.

```js
this._model
             .on('change:duration', this.onDuration, this)
	    .on(...
```